### PR TITLE
Create release page for CASE 0.2.1

### PR DESCRIPTION
**Pull request by plbt5**
* Created new 0.2.1 directory in 'releases/'
* Updated 'releases.yml'

Apparently, modifications have also been made to 'dependencies/CASE-Utilities-Python', but not by me.

### CODE REVIEW

File: _data/releases/releases.yml [hunk 1]
```diff
@@ -4,6 +4,9 @@
 - title: 0.2.0
   date: 2020-08-20
   desc: Release of the CASE ontology with an encoded relationship with UCO. Removed prototype file, imported UCO, and provided investigation namespace.
+- title: 0.2.1
+  date: 2022-04-01
```
Comment (ajnelson-nist) [inline _data/releases/releases.yml:8]: This release hasn't been finished yet, so its date should be `TBD`.

File: releases/0.2.1/index.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+title: 0.2.1 Release
+layout: releases
+custom_css: releases
+---
+
+# {{ page.title }}
+
```
Comment (ajnelson-nist) [inline releases/0.2.1/index.md:8]: @plbt5 , were you intending to list the first 0.2.1-specific proposal on this page?

Comment (ajnelson-nist) [reply releases/0.2.1/index.md:8]: Change Proposal 38 was applied in the history of the CASE `master` branch.  Can you add that to the 0.2.1 and 0.4.0 pages?  We apparently missed listing it as a change in 0.4.0.

**Review by ajnelson-nist — CHANGES_REQUESTED** [review]
Also, the submodule pointer update should be reverted.

**Comment by ajnelson-nist** [comment]
I've corrected the submodule issue.

**Review by regriff — APPROVED** [review]

**Review by regriff — APPROVED** [review]

**Review by ajnelson-nist — APPROVED** [review]

**Pull request merged**